script/lib/git.py

```diff
@@ -307,17 +307,19 @@ def export_patches(repo, out_dir, patch_range=None, dry_run=False):
     # exported patch differs from what exists. Report number of mismatched
     # patches and fail if there's more than one.
     patch_count = 0
+    bad_patches = []
```

Comment:
_nit_ we don't need `patch_count` anymore; it can be replaced with `len(bad_patches)`